### DISCUSSION TITLE
Release 1.3.0

### DISCUSSION THREAD

**Discussion by IsaacMarovitz** [Whisky-App/Whisky#372] [Announcements]
**Quality of Life:**
- Program icons that were previously missing will now display
- `.exe` will display their icon system-wide (i.e. in Finder)
- Locale improvements

**New Features:**
- You can now create shortcuts to programs in your Applications folder
- If Whisky is set as the default program, you can now double-click to open `.exe`, `.msi`. and `.bat` files in your bottles

**Full Changelog**: https://github.com/Whisky-App/Whisky/compare/v1.2.2...v1.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Whisky-App/Whisky/releases/tag/v1.3.0'>Release 1.3.0</a>.</em>

**Comment by Mavix123** [comment]
I have a problem with Whisky 1.3.2 and UltraView Monitor LG 38GL950 with resolution 3840x1600. 
When i started game in macbook display game is okay, but when i connect external monitor do macbook then games not started.
